src/devices/wholeBodyDynamics/WholeBodyDynamicsDevice.cpp

```diff
@@ -508,6 +523,38 @@ bool WholeBodyDynamicsDevice::openExternalWrenchesPorts(os::Searchable& config)
     return ok;
 }
 
+bool WholeBodyDynamicsDevice::openFilteredFTPorts(os::Searchable& config)
+{
+    bool ok = true;
+    if (streamFilteredFT){
+        // create port names
+
+        std::string sensorName;
+        std::string portName;
+        outputFTPorts.resize(estimator.sensors().getNrOfSensors(iDynTree::SIX_AXIS_FORCE_TORQUE));
+        for(int ft=0; ft < estimator.sensors().getNrOfSensors(iDynTree::SIX_AXIS_FORCE_TORQUE); ft++ )
+        {
+            sensorName= estimator.sensors().getSensor(iDynTree::SIX_AXIS_FORCE_TORQUE,ft)->getName() ;
+
+            yDebug() << "wholeBodyDynamics: creating port names for  " << sensorName;
+
+            portName=(portPrefix+"/filteredFT/"+sensorName);
+            //std::unique_ptr<yarp::os::BufferedPort<yarp::sig::Vector>> outputPort
+            outputFTPorts[ft]=std::make_unique<yarp::os::BufferedPort<yarp::sig::Vector>>();
```

Comment:
it's better to leave a space between `>>` in such situations to differentiate between the stream operators.

like,
`std::make_unique<yarp::os::BufferedPort<yarp::sig::Vector> >()`